### PR TITLE
Update sick leave protocol

### PR DESCRIPTION
**Pull request by Vikkigill**
### What 

- Fixed some grammar
- Added more explicit guidance around how to book sick leave 
- Added guidance around how the people team should be notified  

### Why

- Lack of guidance around sick leave protocol

### CODE REVIEW

**Comment by yaseminercan** [comment]
Can we add the CharlieHR bit to first day of absence and doing that plus informing team first thing in the morning?

File: guides/welfare/sick_leave.md [hunk 1]
```diff
@@ -6,18 +6,20 @@ Often when we think of sickness we think of colds, stomach bugs, and perhaps eve
...
+* Don't be a martyr. If you're sick, take the time off. We'd rather you stayed at home than come into the office and infect other people.
 
-* We can provide support if you need help with your [state of mind](state_of_mind.md)
+* We can provide support if you need help with your [state of mind](state_of_mind.md).
 
-* If you need assistance with your sickness you can use our [employee assistance programme](employee_assistance.md)
+* If you need assistance with your sickness, you can use our [employee assistance programme](employee_assistance.md).
 
-* On your first day off, send an email to your team, or the operations team if you are on the Chalet, letting them know that you'll be away and what you had planned for that day. Aim to let them know as early as possible, so they can factor your absence into their tasks for the day.
+* On your first day off, let your team know (Slack, email, etc.), or the operations team if you are on the Chalet, that you'll be away and what you had planned for that day. Aim to let them know as early as possible, ideally by 10:30 AM, so they can factor your absence into their tasks for the day.
```
Comment (chrisblackburn) [inline guides/welfare/sick_leave.md:15]: Should this be start of business over 10.30am? I'd be worried if someone on my team didn't make contact until 10.30

File: guides/welfare/sick_leave.md [hunk 2]
```diff
@@ -6,18 +6,20 @@ Often when we think of sickness we think of colds, stomach bugs, and perhaps eve
...
-* If you are taking four or more days sick leave, please supply a doctors note.
+* If you are taking four or more days sick leave, please supply a doctor's note.
 
-* Record your sick leave in Harvest and CharlieHR
+* Record your sick leave in Harvest (as Absence) and in CharlieHR, for every day that you are on sick leave.
 
-* If you accrue five or more days of sick leave annually, then the people team will check in with you to find out how we can best support you. We want to ensure that we aren't putting you under too much pressure, or that you are unhappy with your current role.
+* The Account Team will have to report you as absent without reason to the People Team by 10:30 AM if we haven't heard from you. They will then get in touch, and if unable to get a hold of you, will call your emergency contact at 11:30 AM.
+
+* If you accrue five or more days of sick leave annually, then the People Team will check in with you to find out how we can best support you. We want to ensure that we aren't putting you under too much pressure, or that you are unhappy with your current role.
```
Comment (chrisblackburn) [inline guides/welfare/sick_leave.md:25]: I think the last sentence reads as: we're checking that you ARE unhappy with your role

**Comment by yaseminercan** [comment]
We have to hear from someone by 10AM. By 10:30 its escalated to People Team who takes over trying to make contact. Maybe we can make that clearer with times?

**Review by adrianclay — APPROVED** [review]
:shipit:

**Pull request merged**